pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -755,3 +761,124 @@ BEGIN
     END LOOP;
 END;
 $$;
+
+CREATE OR REPLACE FUNCTION SCHEMA_PROM.is_stale_marker(value double precision)
+RETURNS BOOLEAN
+AS $func$
+    SELECT float8send(value) = '\x7ff0000000000002'
+$func$
+LANGUAGE SQL IMMUTABLE PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION SCHEMA_PROM.is_normal_nan(value double precision)
+RETURNS BOOLEAN
+AS $func$
+    SELECT float8send(value) = '\x7ff8000000000001'
+$func$
+LANGUAGE SQL IMMUTABLE PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION SCHEMA_PROM.get_label_value(
+        label_id INT)
+    RETURNS TEXT
+AS $$
+    SELECT
+        value
+    FROM SCHEMA_CATALOG.label
+    WHERE
+        id = label_id
+$$
+LANGUAGE SQL STABLE PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.get_label_key_column_name(label_key text, id int)
```

Comment:
yes this calls SCHEMA_CATALOG.pg_name_unique(label_key, id);

Reply:
Do we have a register somewhere of all the unique names we've created? with the original name? I feel like I'd need that if I were querying in sql and had a few random tag names with their names changed...

Reply:
we don't have a register but the name generation is deterministic. Maybe we should record the name though in case we ever change the name generation function